index.js

```diff
@@ -206,7 +206,12 @@ module.exports = input => {
 		}
 	}
 
-	if (check([0x0, 0x0, 0x0, 0x14, 0x66, 0x74, 0x79, 0x70])) {
+	if (check([0x0, 0x0, 0x0, 0x14, 0x66, 0x74, 0x79, 0x70, 0x71, 0x74, 0x20, 0x20]) ||
+		(buf[4] === 0x66 && buf[5] === 0x72 && buf[6] === 0x65 && buf[7] === 0x65) ||
+		((buf[4] === 0x66 && buf[5] === 0x74 && buf[6] === 0x79 && buf[7] === 0x70) &&
+		(buf[8] === 0x71 && buf[9] === 0x74 && buf[10] === 0x20 && buf[7] === 0x20)) ||
+		(buf[4] === 0x6D && buf[5] === 0x64 && buf[6] === 0x61 && buf[7] === 0x74) ||
+		(buf[4] === 0x77 && buf[5] === 0x69 && buf[6] === 0x64 && buf[7] === 0x65)) {
```

Comment:
Rebase against master and you can now use the `{offset}` option in `check()` to handle these cases too.